auth-api/src/auth_api/services/membership.py

```diff
@@ -317,12 +318,14 @@ def _add_or_remove_group(model: MembershipModel):
         """Add or remove the user from/to account holders / product keycloak group."""
         if model.membership_status.id == Status.ACTIVE.value:
             KeycloakService.join_account_holders_group(model.user.keycloak_guid)
-        elif (
-            model.membership_status.id == Status.INACTIVE.value
-            and len(MembershipModel.find_orgs_for_user(model.user.id)) == 0
-        ):
+            if model.org.type_code == OrgType.MAXIMUS_STAFF.value:
+                KeycloakService.add_user_to_group(model.user.keycloak_guid, GROUP_GOV_ACCOUNT_USERS)
```

Comment:
please create group in keycloak and add the roles you need

Reply:
As discussed, investigate the access differences we got from Jia and whether need to add more roles / update auth-api check on some of those dashboard searches that are different for Maximus.

Reply:
resolving this thread. Initial keycloak group created, I will take care of tuning it.